test/specs/rules/validate-indentation.js

```diff
@@ -156,6 +156,23 @@ describe('rules/validate-indentation', function() {
         ]);
     });
 
+    it('should not report errors for using tabs when validateIndentation is a number', function() {
+        checker.configure({ validateIndentation: 4 });
```

Comment:
@kylepaulsen so it's the same that there only one rule - "no rules at all". In your case you want jscs to allow mixed tabs at document, I can't agree with this, when I open some js document I have hundreds of errors for all rules, not just for whitespace. Every time you should ignore jscs, or fix js document.